checks/evaluation/vulnerabilities.go

```diff
@@ -31,20 +32,25 @@ func Vulnerabilities(name string, dl checker.DetailLogger,
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	score := checker.MaxResultScore
-	IDs := []string{}
+	aliasVulnerabilities := []grouper.IDAliases{}
 	for _, vuln := range r.Vulnerabilities {
-		IDs = append(IDs, vuln.ID)
-		score--
+		aliasVulnerabilities = append(aliasVulnerabilities, grouper.IDAliases(vuln))
 	}
 
+	IDs := grouper.Group(aliasVulnerabilities)
+	score := checker.MaxResultScore - len(IDs)
+
 	if score < checker.MinResultScore {
 		score = checker.MinResultScore
 	}
 
 	if len(IDs) > 0 {
+		outputIDs := []string{}
+		for _, v := range IDs {
+			outputIDs = append(outputIDs, strings.Join(v.IDs, " / "))
```

Comment:
how about providing a `warn` message for each vuln? This would also make it nicer for the GitHub dashboard: each warn is reported as a standalone alert the user can fix independently.